i18n/en.json

```diff
@@ -1953,15 +1953,14 @@
   "center_panel.direct.closeDirectMessage": "Close Direct Message",
   "center_panel.direct.closeGroupMessage": "Close Group Message",
   "change_url.close": "Close",
-  "change_url.endWithLetter": "URLs must end with a lowercase letter or number.",
-  "change_url.helpText": "You can use lowercase letters, numbers, dashes, and underscores.",
```

Comment:
You are removing this string here and I guess you are renaming it to something else. I think the main problem here is, if you want to change the ids, you need to go to the source code.

Reply:
Thank you! I commented above and the comment applies to this too. :)